### DISCUSSION TITLE
Item in center position in div tag

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#12010] [Help] [answered]
Hi Team

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/7fbc26df-1e36-43cb-a756-9096b0055d3c)

I want to item expected as shown in Fig 1st.

But my output is as in the below fig. 

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/473ac204-ae7b-4128-8e1b-11e5327b6370)

What can I do?

My code:- 
![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/907a8038-fcec-4f4e-b7e1-cc5121ebda41)

Output as expected:-

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/b423eced-2540-497b-9310-4b57f56e3d68)

Thanks 
Minal P

**Comment by wongjn** [comment]
Consider applying `justify-content: center` via the `justify-center` class to the `<nav>` element.

---

As an aside, [please consider not uploading images of code/data/errors](https://meta.stackoverflow.com/q/285551/19655631). Instead, consider using [Tailwind Play](https://play.tailwindcss.com) for providing Tailwind code snippets.

**Reply by MinalPatilRPA** [reply]
when I apply the justify-content: center class to the nav element it will get output as shown in the below fig.

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/3764b462-6809-4c03-b02f-798d494e69ca)

Excepted output is:-

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/78e7b0ba-8069-4004-9678-ee3f8cd44fe1)

Note:- When hovering over the option it will be gray.

**Reply by wongjn** [reply, marked as answer]
I'm sorry, I misinterpreted your requirements. Consider applying `width: 100%` via `w-full` to the `<nav>` element and then applying `flex: 1 1 0%` to each of the `<div>` children via `flex-1`. You may then need to center the `<a>` elements.
